docs/_scripts/add_content_repo.sh

```diff
@@ -1,14 +1,7 @@
 # Add created PythonPackage content to repository
 echo "Add created Python Package to repository"
-export TASK_URL=$(http POST $BASE_ADDR$REPO_HREF'modify/' \
-  add_content_units:="[\"$PACKAGE_HREF\"]" | jq -r '.task')
-
-# Poll the task (here we use a function defined in docs/_scripts/base.sh)
-wait_until_task_finished $BASE_ADDR$TASK_URL
+pulp python repository add --name foo --filename "shelf-reader-0.1.tar.gz"
```

Comment:
This is a little unintuitive, we're saying ```repository add --name foo``` but it's not creating a repository, it's adding content *to* a repository.

Does a reverse order work?  ```pulp python repository --foo add --filename "shelf-reader-0.1.tar.gz"```

Or maybe the names could be changed slightly?

```pulp python repository add-content --repo-name foo --filename "shelf-reader-0.1.tar.gz"```